packages/client3/pages/artist/dashboard/index.page.tsx

```diff
@@ -31,17 +39,18 @@ const DummyDashboardDataEmpty = {
   },
 };
 
-function DashBoard() {
-  const [dashBoardData, setDashBoardData] = useState<
+function Dashboard() {
+  const [dashBoardData, setDashboardData] = useState<
     typeof DummyDashboardDataEmpty
   >(DummyDashboardDataEmpty);
 
   const [isVerified, setIsVerified] = useState(false);
   const user = useUser();
 
   useEffect(() => {
-    const isVerified = hasUid(user);
+    const isVerified = true; //hasUid(user);
```

Comment:
probably change this back to `hasUid(user)`

Reply:
fixed